src/sql/workbench/contrib/notebook/browser/cellViews/outputArea.css

```diff
@@ -20,3 +20,7 @@ output-area-component .notebook-output {
 	white-space: pre-wrap;
 	word-wrap: break-word;
 }
+
+.output-userselect {
```

Comment:
Can you scope this down farther? Under something like notebook-output would be great.